arcconfig/arcconfig.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package arcconfig defines the API Runtime Control config and provides
+// the config loading and validation functions.
+package arcconfig
```

Comment:
Is `config` already taken? The `arc` prefix does not seem to add any more specificity

Reply:
`config` is still available. There is a config.go file in the `server` pkg though. FYI, I made this separate package because I thought it would be simpler to import it.

Reply:
We may want to refactor/rename a little. We have a global "config" in the server package.

Reply:
I see... this is fine the way it is then, we can always refactor it later without breaking the contract with clients.